doc/developer-notes.md

```diff
@@ -228,7 +228,7 @@ Development guidelines
 A few non-style-related recommendations for developers, as well as points to
 pay attention to for reviewers of Bitcoin Core code.
 
-General Bitcoin Core
+General Dogecoin Core
```

Comment:
nit: we haven't really changed anything here so technically making this "Dogecoin Core" documentation would be a greater effort. Interested in other opinions on this.

Reply:
i think some changes, especially backward compatible changes and guidlelines should be added to the documentation for devs, so they dont have to write you emails asking which pieces need to stay ;]

Reply:
Sure. But let's separate that from this then.

Reply:
well its more similar to litecoin than bitcoin at this point? (https://github.com/litecoin-project/litecoin/blob/master/doc/developer-notes.md)

Reply:
Not really.

Reply:
well  you have said yourself that its diverged a lot from bitcoin core. anyway.

Reply:
i sent you an email but you never got back to me, cheers

Reply:
How about changing it to "General Code" so it's more... general? I have no strong feelings, but this heading is more parallel to other headings in the document.